.gitignore

```diff
@@ -6,6 +6,7 @@ target
 integration_tests/dist_query/dist-query-testing
 integration_tests/dist_query/tsbs
 integration_tests/dist_query/output
+integration_tests/ceresmeta/
```

Comment:
This file is not used any more, you can just delete this file.

In newer version, meta will be generated in `target` dir.